requirements_dev.txt

```diff
@@ -1,12 +1,13 @@
-click
-numpy
-pyyaml
-cffi
-scipy
+21cmFAST>=3.0.0dev
```

Comment:
The syntax of this is slightly different than before, but I'm going to assume it's right.
I see that you added and removed the 21cmFAST @ git line.

Reply:
Yes, this is something we should be doing -- that is, only relying on the most obvious _publicly available_ release of 21cmFAST. Installing from GitHub means we'll always get all the latest bits and pieces that haven't been formally released yet.